### PR TITLE
chore(release): 3.0.0-beta.0

### PR DESCRIPTION
**Pull request by ericdeansanchez**

### CODE REVIEW

**Comment by ericdeansanchez** [comment]
Looking at this again, I'm wondering if I should drop `bower.json` in a different PR... Let me know what ya'll think.

File: dist/imgix-core-js.cjs.js [hunk 1]
```diff
@@ -149,7 +149,7 @@ function _nonIterableRest() {
 }
 
 // package version used in the ix-lib parameter
-var VERSION = '2.3.2'; // regex pattern used to determine if a domain is valid
+var VERSION = '3.0.0-beta.0'; // regex pattern used to determine if a domain is valid
 
 var DOMAIN_REGEX = /^(?:[a-z\d\-_]{1,62}\.){0,125}(?:[a-z\d](?:\-(?=\-*[a-z\d])|[a-z]|\d){0,62}\.)[a-z\d]{1,63}$/i; // minimum generated srcset width
```
Comment (sherwinski) [inline dist/imgix-core-js.cjs.js:155]: I couldn't help but notice that the comments here (L152-L160) are all shifted up by one. I'm sure that it's being outputted this way by rollup so there might not be anything actionable on our end.

**Review by sherwinski — CHANGES_REQUESTED** [review]
> Looking at this again, I'm wondering if I should drop bower.json in a different PR... Let me know what ya'll think.

I agree that it should have its own PR.
Also I noticed that this PR is missing the `package-lock.json` file.

**Comment by frederickfogerty** [comment]
What's the breaking change here?

**Comment by ericdeansanchez** [comment]
> What's the breaking change here?
@frederickfogerty okay, good, I think this means we agree then. I'll send you the convo from yesterday. 

I'm closing this one for now.

**Pull request closed without merging**